src/main.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
```

Comment:
let's rename `src` to some meaningful module name
If you want to make module callable like `python -m your_module --arg1 arg1 --arg2 arg2 ...` - you can rename `main.py` to `__main__.py`